.github/.cspell/dictionary.txt

```diff
@@ -0,0 +1,28 @@
+bindgen
+clippy
+codecov
+condvar
+consts
+deque
+drawio
+funcs
+kajita
+lcov
+libglu
+nalgebra
+nullspace
+println
+rctree
+rgba
+robo
+rustdocflags
+rustflags
+rustfmt
+rustup
+ryzen
+serde
+struct
```

Comment:
struct is not English? So sorry but could you double check? I think you are right basically.

Reply:
I think "struct" is a short form for "structure". It may not have been formal English.

Anyway, I removed it from the dictionary in the same way as "println".